modules/infinite_odoscope/js/models/odoscope-article-model.js

```diff
@@ -31,20 +34,23 @@
       /**
        * Set latest Model
        */
-      tmpModel = this.get('list').shift();
+      latestModel = this.get('list').shift();
```

Comment:
Just a note:

I find it confusing that this.get('list').shift() returns an actual model. But it's okay for now.